packages/core/upload/admin/src/utils/tests/updateUrlForCaching.test.js

```diff
@@ -0,0 +1,31 @@
+import { updateUrlForCaching } from '..';
+
+describe('updateUrlForCaching', () => {
+  const urlString = 'https://example.com/';
+  const updateTime = '2023-07-19T03:00:00.000Z';
+
+  test('returns null for null url string', () => {
+    expect(updateUrlForCaching(null, undefined)).toBeNull();
+  });
+
+  test('throws error for null url string with updateAt', () => {
+    expect(() => updateUrlForCaching(null, updateTime)).toThrowError();
+  });
```

Comment:
i'm not sure about this, imo i think we'd be better off returning `null` again, @gu-stav what do you think?

Reply:
Yeah, I wouldn't expect this to be an error when the other null case returned null. I have updated the code.